suite-native/module-send/src/sendOutputsFormSchema.ts

```diff
@@ -194,32 +194,49 @@ export const sendOutputsFormValidationSchema = yup.object({
             }),
         )
         .required(),
-    isRippleDestinationTagEnabled: yup.boolean(),
-    rippleDestinationTag: yup
+    isDestinationTagEnabled: yup.boolean(),
+    destinationTag: yup
         .string()
-        .when('isRippleDestinationTagEnabled', {
+        .when('isDestinationTagEnabled', {
             is: true,
             then: schema => schema.required('Destination Tag is required'),
             otherwise: schema => schema.notRequired(),
         })
-        .matches(/^\d*$/, 'You can only use positive numbers for the destination tag.')
+        .test(
+            'is-destination-tag-a-number',
+            'You can only use positive numbers for the destination tag.',
+            (value, { options: { context } }: yup.TestContext<SendFormFormContext>) => {
+                const { symbol } = context!;
+
+                if (!symbol) return true;
+                const networkType = getNetworkType(symbol);
+                if (networkType !== 'ripple') return true;
```

Comment:
nit: I would reverse the condition to be future proof in case we add something else with destination tag
```suggestion
                if (networkType === 'stellar') return true;
```

Reply:
1ccdb5cf18c2430572b3f56fd04a49f7068778d5